### PR TITLE
Feat/campaign screen

### PR DESCRIPTION
**Pull request by ashishiiit01**
Fixed Height Error Campaign screen

### CODE REVIEW

File: screens/CampaignScreen/index.js [hunk 1]
```diff
@@ -1,22 +1,95 @@
...
+                    if (route.name === 'Home') {
+                        iconName = isFocused
+                            ? 'home'
+                            : 'home';
+
+                    } else if (route.name === 'Message') {
+                        iconName = isFocused ? 'message' : 'message';
+                    }
+                    else if (route.name === 'Settings') {
+                        iconName = isFocused ? 'view-comfortable' : 'view-comfortable';
```
Comment (theankurkedia) [inline screens/CampaignScreen/index.js]: Both the conditions are the same

File: screens/CampaignScreen/index.js [hunk 2]
```diff
@@ -1,22 +1,95 @@
...
+                    }
+                    else {
+                        iconName = isFocused ? 'supervised-user-circle' : 'supervised-user-circle';
```
Comment (theankurkedia) [inline screens/CampaignScreen/index.js]: Here again

File: screens/CampaignScreen/Content.js [hunk 3]
```diff
@@ -1,52 +1,29 @@
...
-                            </Box>
-                        ))
-                    }
-                </Box>
-            </Box>
-        </ScrollView >
+        <Box
+            border={'#333166'}
+            bg={'bg.500'}
+            minHeight={Platform.OS === 'ios' ? DEVICE_HEIGHT - 300 : DEVICE_HEIGHT - 230}
```
Comment (theankurkedia) [inline screens/CampaignScreen/Content.js:17]: Why do we need DEVICE_HEIGHT?

File: screens/CampaignScreen/index.js [hunk 4]
```diff
@@ -1,22 +1,95 @@
...
+
+                    return (
+                        <TouchableOpacity
+                            key={index}
+                            accessibilityRole="button"
+                            accessibilityStates={isFocused ? ['selected'] : []}
+                            accessibilityLabel={options.tabBarAccessibilityLabel}
+                            testID={options.tabBarTestID}
+                            onPress={onPress}
+                            onLongPress={onLongPress}
```
Comment (theankurkedia) [inline screens/CampaignScreen/index.js]: Why do we need onLongPress?

File: screens/CampaignScreen/index.js (hunk 1 above)
Comment (ashishiiit01) [reply screens/CampaignScreen/index.js]: I have removed this condition. There is no use.

File: screens/CampaignScreen/index.js (hunk 2 above)
Comment (ashishiiit01) [reply screens/CampaignScreen/index.js]: removed it

File: screens/CampaignScreen/index.js (hunk 4 above)
Comment (ashishiiit01) [reply screens/CampaignScreen/index.js]: I got this code from some site to design tabs, but onLongPress has no use. I have removed it also.

File: screens/CampaignScreen/Content.js (hunk 3 above)
Comment (ashishiiit01) [reply screens/CampaignScreen/Content.js:17]: I am using DEVICE_HEIGHT to give the min-height to middle content section, Without height, flex is not working as expected.

**Pull request merged**